users/forms.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
These forms are not being used ATM. So we should just remove this file.

Reply:
Done